prime-router/src/main/kotlin/azure/WorkflowEngine.kt

```diff
@@ -62,7 +63,8 @@ class WorkflowEngine(
      * Place a report into the workflow
      */
     fun dispatchReport(nextAction: Event, report: Report, txn: Configuration? = null) {
-        val (bodyFormat, bodyUrl) = blob.uploadBody(report)
+        val forceFormat = if (nextAction.eventAction == Event.EventAction.BATCH) OrganizationService.Format.CSV else null
+        val (bodyFormat, bodyUrl) = blob.uploadBody(report, forceFormat)
```

Comment:
Batches should always be CSV formatted.

Reply:
What about future incoming hl7 data?    Do we really want to transform that to CSV, then back to HL7?